source/VersionHandlerImpl/src/FileUtils.cs

```diff
@@ -288,5 +288,26 @@ internal class FileUtils {
             }
             return NormalizePathSeparators(searchDirectory);
         }
+
+        /// <summary>
+        /// Checks out a file should Version Control be active and valid.
+        /// </summary>
+        /// <param name="path">Path to the file that needs checking out.</param>
+        /// <returns>False should the checkout fail, otherwise true.</returns>
+        public static bool CheckoutFile(string path)
+        {
+            if (UnityEditor.VersionControl.Provider.enabled && UnityEditor.VersionControl.Provider.isActive &&
+                (!UnityEditor.VersionControl.Provider.requiresNetwork ||
+                 UnityEditor.VersionControl.Provider.onlineState == UnityEditor.VersionControl.OnlineState.Online))
+            {
+                var task = UnityEditor.VersionControl.Provider.Checkout(path,
```

Comment:
I'm not sure how this fixes https://github.com/googlesamples/unity-jar-resolver/issues/262 the code looks exactly the same which means it's possible we could end up with a MissingMethodException.  I'll merge and try working around this.